openedx/core/djangoapps/credit/tests/test_verification_access.py

```diff
@@ -80,8 +80,9 @@ def test_creates_user_partitions(self):
         self._update_partitions()
 
         # Check that a new user partition was created for the ICRV block
-        self.assertEqual(len(self.course.user_partitions), 1)
-        partition = self.course.user_partitions[0]
+        partitions = _get_non_enrollment_track_partitions(self.course)
```

Comment:
Note to reviewers: the VerificationUserPartition will be going away soon, but in the meantime I have to keep tests passing!